### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jospoortvliet**
let's add a note about this issue pls... It is easy to work around but then ppl should know ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,4 @@ Once configured, the folders will show up in the home folder for each user in th
 ## A note about encryption
```
Comment (MariusBluem) [inline README.md]: So it’s not longer a note about encryption? 😅

**Comment by jospoortvliet** [comment]
Whaddayathunk?

File: README.md (hunk 1 above)
Comment (jospoortvliet) [inline README.md]: :see_no_evil:

**Comment by MariusBluem** [comment]
:+1:

**Pull request merged**